docs/notebooks/qaoa-transpiler.ipynb

```diff
@@ -0,0 +1,800 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Transpile QAOA circuits with a best SWAP strategy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this comprehensive guide, we will walk you through the **Quantum Approximate Optimization Algorithm (QAOA)** workflow using **[Qiskit Patterns](https://www.ibm.com/quantum/blog/qiskit-patterns).**\n",
+    "We'll demonstrate the application of these patterns specifically within the domain of **combinatorial optimization** and showcase how the QAOA, a hybrid quantum-classical iterative method, can be employed effectively.\n",
+    "\n",
+    "- **Step 1**: Formulating a combinatorial in terms of finding the ground state of a Hamiltonian. This reformulated problem can be understood by a quantum computer.\n",
+    "- **Step 2**: Preparation of the necessary quantum circuits optimized for execution on quantum hardware.\n",
+    "- **Step 3**: Iteratively utilizing Qiskit's `Sampler` primitive to draw samples from the prepared quantum circuits in Step 2. These samples will inform the loss function of our algorithm routine.\n",
+    "- **Step 4**: Converting the samples from Step 3 into a solution for our combinatorial optimization problem.\n",
+    "\n",
+    "Additionally, We will demonstrate how to build and optimize QAOA circuits using the **SWAP strategy**, a specifically designed for QAOA, for a specific instance of the **Max-Cut**."
```

Comment:
```suggestion
    "Additionally, We will demonstrate how to build and optimize QAOA circuits using the **SWAP strategy**, a specifically designed transpiler pass for QAOA applied to quadratic problems."
```
I think that you can remove the MaxCut reference here and replace by quadratic problem which is more general.